apps/browser-extension-wallet/src/utils/validators/address-book.ts

```diff
@@ -75,6 +92,45 @@ export const validateWalletHandle = async ({ value, handleResolver }: validateWa
   return '';
 };
 
+type hasHandleOwnerChangedArgs = {
+  value: string;
```

Comment:
```suggestion
  handleResolution: HandleResolution;
```
Then you can omit `address`

Reply:
I'd favour not passing all the information when we don't need it. But ok.